configs/components/puppet-ca-bundle.rb

```diff
@@ -4,14 +4,41 @@
   pkg.build_requires "openssl"
```

Comment:
Note: I'm not able to comment on it directly, but it looks like puppet-agent is using version 1.0.8 of the puppet-ca-bundle (in configs/components/puppet-ca-bundle.json) on all three puppet-agent branches.

Reply:
Hmm, this is a conundrum - the puppet-ca-bundle component version is loaded from json in both the runtime (v1.0.7) and the existing agent (v1.0.8) - Does it make sense to have multiple json files for the component and choose one based on the project? @underscorgan do you have ideas?

Reply:
I suspect the difference is just drift from when the bundle was updated in September, we should update the json to have version 1.0.8

Reply:
Updated to 1.0.8, thanks both